### DISCUSSION TITLE
Where are global functions?

### DISCUSSION THREAD

**Discussion by kehrli** [iNavFlight/inav#9255] [Q&A]
I'm using iNav 6.1 with a SpeedyBee F405 WING controller.  I type "gf" in the CLI and I get "### ERROR: Unknown command, try 'help'".  I see that it is a tab in Programming on a Pawel video, but I don't have that tab.  Is this because SpeedyBee doesn't support global functions?  Or has it moved to somewhere new?

**Comment by DzikuVx** [comment]
https://github.com/iNavFlight/inav/blob/master/docs/Programming%20Framework.md
